### ISSUE TITLE
Notify used before downloading integrated python

### ISSUE THREAD

**Issue by joeyballentine** [joeyballentine/chaiNNer#926]
**Motivation**
Many people do not want to use the integrated python, plus it tends to fail to download. 

**Description**
We should have a message like "Integrated python not found. Would you like to download?" which would allow them to bypass it and turn "use-system-python" on. Then we should also notify after they click that saying "chainner will now install it's required dependencies to your system python installation, would you like to proceed?" With an option to switch back to integrated, just in case.

We also should detect when integrated python fails to download, and show an appropriate error message with a retry option. 

Basically we should make it optional but still sorta suggest that the best option is to use integrated, but then also make it so downloading integrated is less likely to cause confusion.

Also, we need to check if the python version is undefined and warn the user that the python instance was corrupt or not found, or something.

**Alternatives**
At the very least we should warn the user that it failed to download properly rather than silently failing and giving them a cryptic error with no relevant logs.

**Comment by RunDevelopment** [comment]
I don't think we should do this. To understand the question "Would you like to download integrated Python?", users have to know at least the following things:

1. That chainner needs a python installation to function.
2. Whether they can even use the alternative python environment (not every one has python installed).
3. That integrated and system are the only choices. Not having a python installation isn't possible.

I do not want to give new users this question. A new user shouldn't have to know what integrated Python. I mean, they shouldn't even have to know that we use Python in the backend at all. Installing a program shouldn't have to require knowing how that program works.

> Many people do not want to use the integrated python, plus it tends to fail to download.

The UX problem of the download failing has been solved by #2768. Advanced users that do not want to use the integrated Python can install chainner normally (with integrated) and then switch to their System python in the settings.

**Comment by joeyballentine** [comment]
I completely agree. I never really wanted to do this, it was just requested a lot
